geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/StatsBasedEstimator.scala

```diff
@@ -242,7 +244,9 @@ class CountEstimator(sft: SimpleFeatureType, stats: GeoMesaStats) extends LazyLo
         bounds.bounds.map(_.bounds)
       }
       extractedBounds.flatMap { bounds =>
-        if (bounds.contains((None, None))) {
+        if (bounds.isEmpty) {
+          Some(0L) // disjoint range
+        } else if (bounds.exists(_ == (None, None))) {
```

Comment:
haha, yeah... i was trying not to change the method signature, but I should probably update it at this point